app/views/devise/sessions/new.html.erb

```diff
@@ -35,7 +20,20 @@
       </li>
     <% end %>
 
-    <li role="separator" class="mdc-deprecated-list-divider"></li>
-    <%= render "devise/shared/links" %>
-  </ul>
+    <p class="text-center text-sm/6 text-neutral-500">
+      ¿No estás registrado?
+
+      <%= link_to "Registrarte",new_registration_path(resource), class: "font-semibold text-neutral-700 hover:text-indigo-700" %>
+    </p>
+
+    <p class="text-center text-sm/6 text-neutral-500">
+      ¿Has olvidado tu contraseña?
+
+      <%= link_to "Restablecer contraseña", new_password_path(resource), class: "font-semibold text-neutral-700 hover:text-indigo-700" %>
+    </p>
```

Comment:
Siento que movería esto para la card donde están los inputs y el botón, entre medio de ellos. Incluso podríamos [revisitar la discusión que tuvimos en #720](https://github.com/cedarcode/mi_carrera/pull/720?notification_referrer_id=NT_kwDOAsNPiLQxNjE3NTM4ODEyMTo0NjM1NDMxMg#discussion_r2082164992).

De hecho, siento que pondría todo lo relacionado con el log in, las distintas formas de loguearse, etc adentro de la card y dejaría afuera solo el link para registrarte si no tenés cuenta. Qué te parece?

De todas formas, tampoco tomaría este comentario como un blocker para este PR

Reply:
Yo estaba hablando del link en el login a la página de forgot you password – de todas formas cómo está ahora me parece bien 🙂